nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/ChartProperties.java

```diff
@@ -36,6 +36,8 @@
  */
 public class ChartProperties {
     private static final Logger log = LoggerFactory.getLogger(ChartProperties.class);
+    public static final int PROJECTION_BORDER_DEFAULT_SIZE = 150;
+    public static final int EMPTY_BORDER_DEFAULT_SIZE = 7;
```

Comment:
7 is the initial value for the border, but it is a bit strange since 7 is not one of the options in the menu size combobox in Attributes -> Style tab. So once you click on the combo box you can never get back to 7. Perhaps we should change it to a default value of 5?

Reply:
makes sense. I don't know how I came up with seven. Unless it just looked good